packages/typespec-ts/test/modularUnit/enumUnion.spec.ts

```diff
@@ -842,13 +842,13 @@ describe("model type", () => {
         modelFile!.getFullText()!,
         `
         export interface Test {
-          color: LR | UD;
+          color: Lr | Ud;
```

Comment:
Could you add a UT here for name `createdByType` both for RLC and Modular?

Reply:
added